doc/examples/nine_tile_box/lib/main.dart

```diff
@@ -25,8 +26,8 @@ class MyGame extends Game {
   @override
   void render(Canvas canvas) {
     const length = 300.0;
-    final x = (size.width - length) / 2;
-    final y = (size.height - length) / 2;
+    final x = (size.x - length) / 2;
+    final y = (size.y - length) / 2;
     nineTileBox.draw(canvas, x, y, length, length);
```

Comment:
this method could be refactored to take in a Vector2. then you can do:

```
final len = Vector2(length, length);
nineTileBox.draw(canvas, (size - len) / 2, len);
```

Reply:
Done, please have a look so I got all the variable names describing the correct thing.